src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -78,7 +79,8 @@ public BaseSwerveSubsystem(
         BaseSwerveModule bottomRightModule,
         double maxVel, double maxAccel, double maxOmega, double maxAlpha,
         SwerveDriveKinematics kinematics,
-        PhotonWrapper photonWrapper
+        PhotonWrapper photonWrapper,
+        LEDSubsystem ledSubsystem
```

Comment:
perhaps we can add a constructor that takes no LEDSubsystem, leave the field as null, and have logic that's resilient to that. to minimize our impact, for example on SwerveSubsystem2020, and make it possible to not have LEDs